src/main/java/pro/sky/homeworkemployee/EmployeeExistExeption.java

```diff
@@ -0,0 +1,12 @@
+package pro.sky.homeworkemployee;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST, reason = "Этот сотрудник уже добавлен")
+public class EmployeeExistExeption extends RuntimeException {
+    public EmployeeExistExeption() {
+        super();
+        System.out.println("Этот сотрудник уже добавлен");
```

Comment:
```suggestion
        super("Этот сотрудник уже добавлен");
```
конструктор не должен выполнять каких-то сторонних действий кроме инициализации состояния. Выводить текст ошибки мы должны в том месте, где данная ошибка будет поймана с помощью try-catch